src/clarity-angular/tabs/vtabs-nav-link.ts

```diff
@@ -0,0 +1,79 @@
+
+import {Component, Input, EventEmitter} from "@angular/core";
+
+/**
+ * @class VTabsNavLink
+ *
+ * @description
+ * A child component of VTabs that:
+ *  1. Creates the navigation links
+ *  2. Exposes an input property that allows binding to preselect a specific tab
+ *  3. Has functionality to allow an instance to become active and emit the change up to VTabs
+ */
+@Component({
+    selector: "clr-vtabs-nav-link",
+    template: `
+        <li (click)="activateLink()" >
```

Comment:
Oh, yes, that's a no-go. Click listener goes on the button, definitely.
And once that's done, you don't want a list item anymore, just the host with the correct classes, the button, and `role="listitem"` on the host for accessibility (with the parent having `role="list"`). I don't believe we do this in horizontal tabs, by the way, we should go back to it at some point.